ament_cmake_export_interfaces/cmake/ament_export_interfaces.cmake

```diff
@@ -19,6 +19,9 @@
 # ``install(TARGETS ... EXPORT name ...)``.
 # The ``install(EXPORT ...)`` invocation is handled by this macros.
 #
+# :param HAS_LIBRARY_TARGET: if set, an environment variable is being defined
```

Comment:
This might be a bit clearer as:

```
:param HAS_LIBRARY_TARGET: if set, an environment variable will be defined
so the library can be found by runtime executables
```

Reply:
My text contains a wrong word: `executables` was probably a left over from previous iterations.

For your suggested text the question would then be: what are "runtime executables"?

Reply:
I was wondering the same thing, I think I mistook the meaning.  How about:

```
:param HAS_LIBRARY_TARGET: if set, an environment variable will be defined
so the library can be found at runtime
```

Reply:
Done in c0075ba0bc55b5ae5d08a3cf01dc4ba1e380dd02.